app.coffee

```diff
@@ -66,6 +69,8 @@ app.post "/webhook", (req, res) ->
 
   if slack.userInfoById(req.body.user_id)
     avatar = slack.userInfoById(req.body.user_id)['profile']['image_72']
+    if process.env.ADMIN_NICK == "full"
+      adminNick = slack.userInfoById(req.body.user_id)['profile']['real_name']
```

Comment:
I think you wanted to change `nick` for the request below. Assigning `adminNick` here won't do any good if you want to send full name in messages.